diagnostic.md

```diff
@@ -6,77 +6,118 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```bash
-// your response here
+
+A backend is used to communicate with the server that is largely responsible for
+handling requests from the front end and sending the appropriate resource that is
+requested.
+
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```bash
-// your response here
+
+Model
+
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```bash
-// your response here
+
+Controller
+
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```bash
-// your response here
+
+Because we will generate the front end as opposed to outputting HTML through the backend.
+
 ```
 
 What does C.R.U.D stand for?
 
 ```bash
-// your response here
+Create
+Read
+Update
+Delete
 ```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```bash
-// your response here
+
+Controller
+
 ```
 List at least 5 standard actions that C.R.U.D corresponds to?
 
 ```bash
-// your response here
+
+GET
+POST
+PATCH/PUT
+DELETE
```

Comment:
Actions are: Index, Show, Create, Update, Destroy. The HTTP requests correlate with those methods in the controller.